index.html

```diff
@@ -1,657 +1,1213 @@
     <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-    <div class="main">
-        <header class="header">
-            <div class="search-bar">
-                <div class="cmp-search-lister__search-box">
-                    <input class="cmp-search-lister__search-box-text" data-cmp-hook-formssearch="input" type="text" placeholder="Search resources...">
-                </div>
-                <span class="close-search">
-                    <svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
-                        <rect x="2" y="1" width="21.861" height="2" transform="rotate(45 2 1)" fill="#000000"/>
-                        <rect x="1" y="16" width="21.861" height="2" transform="rotate(-45 1 16)" fill="#000000"/>
-                    </svg>
-                </span>
-            </div>
-            <div class="container">
-                <div class="site-logo">
-                    <div data-asset-id="" id="" data-cmp-data-layer="" class="logo-wrap" itemscope="" itemtype="">
-                        <a class="logo-link" data-cmp-clickable="" href="">
-                            <img srcset="./assets/logo.svg" src="./assets/logo.svg" loading="lazy" class="logo-img" itemprop="contentUrl"  alt="logo" title="logo">
-                        </a>
-                        <meta itemprop="caption" content="Component">
+    <link rel="stylesheet" href="./aem/styles/grid.min.css">
+
+  </head>
+  <body class="empty-page page basicpage" id="empty-page-3d51119225">
+    <script src="https://global.oktacdn.com/okta-auth-js/7.0.0/okta-auth-js.min.js"></script>
+    <div class="alert alert-danger dn" role="alert"></div>
+    <div class="modal fade sign-in-modal"></div>
+    <div id="okta-config" class="hidden"></div>
+
+    <div class="content-parsys responsivegrid">
+      <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+        <!-- HEADING WRAP -->
+        <div class="container responsivegrid header aem-GridColumn aem-GridColumn--default--12">
+          <div id="header-main" class="cmp-container">
+            <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+              <!-- SEARCH OVERLAY WRAP -->
+              <div class="container responsivegrid home-search-overlay aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                <div id="search-overlay" class="cmp-container">
+                  <!-- CLOSE IMAGE -->
+                  <div class="close">
+                    <div class="cmp-image">
+                      <a href="#" class="close">
+                        <img class="cmp-image__image" src="./assets/close.svg">
+                      </a>
                     </div>
+                  </div>
+                  <!-- SEARCH FORM -->
+                  <form action="/search">

```

Comment:
@wrandall22 Would the [Quick Search](https://www.aemcomponents.dev/content/core-components-examples/library/core-content/search.html) component be the best choice here?

Reply:
We aren't using the quick search component, it is using the Omni-search I think instead, or maybe our old custom search. I forget which one.